src/styles/_catalog.scss

```diff
@@ -1,22 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  @extend %grid-center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  grid-template-columns: repeat(1, $card-width);
 
   &__card {
-    display: flex;
-    justify-content: center;
+    @extend %grid-center;
```

Comment:
Interesting)

Reply:
why?) I use these properties two times in my code so I can make a placeholder, can`t I?

Reply:
You can, it's ok)